src/sonic-config-engine/sonic-cfggen

```diff
@@ -174,6 +182,7 @@ def main():
     parser.add_argument("-d", "--from-db", help="read config from configdb", action='store_true')
     parser.add_argument("-H", "--platform-info", help="read platform and hardware info", action='store_true')
     parser.add_argument("-s", "--redis-unix-sock-file", help="unix sock file for redis connection")
+    parser.add_argument("-i", "--interface", help="print specific port details")
```

Comment:
since you add a new argument, I assume you will add a test for the new argument.

Reply:
Added test for it.